assets/l10n/app_en.arb

```diff
@@ -440,5 +440,13 @@
   "allMessagesPageTitle": "All messages",
   "@allMessagesPageTitle": {
     "description": "Title for the page of all messages"
+  },
+  "groupDMNotificationMessage": "{senderFullName} to you and {num, plural, =1{1 other} other{{num} others}}.",
```

Comment:
```suggestion
  "notifGroupDmConversationLabel": "{senderFullName} to you and {num, plural, =1{1 other} other{{num} others}}.",
```

* In camel-cased identifier names, write acronyms like `…Dm…`, not `…DM…`.
* For this file which has a big list of translatable strings from across the whole app, giving things a prefix that relates to what part of the app it's about is helpful for organization.
* "Message" is confusing here, because it sounds like it's about the Zulip message that someone sent and caused the notification.

Reply:
Separately on this line:

This added a period at the end of the string. That isn't in the previous version, and it doesn't make sense in context where this is used in the UI.

Reply:
Yes, I was going to ask you if I should add a period at the end of the sentence when I got the PR out of draft. Thanks for the comment.